cmd/cli/kubectl-kyverno/commands/apply/command.go

```diff
@@ -70,11 +70,12 @@ func Command() *cobra.Command {
 	var removeColor, detailedResults, table bool
 	applyCommandConfig := &ApplyCommandConfig{}
 	cmd := &cobra.Command{
-		Use:          "apply",
-		Short:        command.FormatDescription(true, websiteUrl, false, description...),
-		Long:         command.FormatDescription(false, websiteUrl, false, description...),
-		Example:      command.FormatExamples(examples...),
-		SilenceUsage: true,
+		Use:           "apply",
+		Short:         command.FormatDescription(true, websiteUrl, false, description...),
+		Long:          command.FormatDescription(false, websiteUrl, false, description...),
+		Example:       command.FormatExamples(examples...),
+		SilenceUsage:  true,
+		SilenceErrors: true,
```

Comment:
Do we need to silence errors ?

Reply:
Not in all cases. 

Here is the behavior I think we want:
1. If there is a policy failure or error in the engine results, exit with a non-zero but do not show any additional error statements.
2. If there is any unexpected error in the CLI command execution, print the error.

Reply:
Fixed